news-bundle/contao/templates/news/news_latest.html5

```diff
@@ -11,9 +11,11 @@
 
   <h2><?= $this->linkHeadline ?></h2>
 
-  <div class="ce_text block">
-    <?= $this->teaser ?>
-  </div>
+  <?php if ($this->teaser): ?>
+    <div class="ce_text block">
+      <?= $this->extractStyleAttributesForCsp($this->teaser) ?>
+    </div>
+  <?php endif; ?>
```

Comment:
This is a slight behavior change. Previously the `<div class="ce_text block">` wrapper was always output in the default template. Now it won't be output if there is no teaser.

Since the `extractStyleAttributesForCsp` method has now been changed to also allow `null` values, I think this can be reverted back to its original state?

Reply:
Yes, that's actually a bug fix. Why would you need an empty div? but yeah, might as well just revert this.

Reply:
I agree, but I don't think we should change it in the context of this PR.